### ISSUE TITLE
member-ordering: Does not work inside object literal

### ISSUE THREAD

**Issue by andy-hanson** [palantir/tslint#1954]
### Bug Report

- __TSLint version__: `4.2.0-dev.0`
- __TypeScript version__: `2.2.0-dev.20161230`
- __Running TSLint via__: CLI

#### TypeScript code being linted

```ts
const o = {
	foo: class C {
		foo() {}
		x = 0;
	}
}

class D {
	foo() {}
	x = 0;
}
```

with `tslint.json` configuration:

```json
{
	"rules": {
		"member-ordering": [true, { "order": "fields-first" }]
	}
}
```

#### Actual behavior

Failure in class D

#### Expected behavior

Failures in both class C and class D
